kubernetes/db/ignite-deployment.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: ignite-master
+  labels:
+    app: ignite
+spec:
+  selector:
+    matchLabels:
+      app: ignite
+      role: master
+      tier: backend
+  replicas: 1
```

Comment:
Do we test multi replicas? I would suggest to test it out and use 3 as the default replica number.